src/main/java/org/elasticsearch/rest/action/admin/indices/analyze/RestAnalyzeAction.java

```diff
@@ -64,4 +73,58 @@ public void handleRequest(final RestRequest request, final RestChannel channel,
         analyzeRequest.charFilters(request.paramAsStringArray("char_filters", analyzeRequest.charFilters()));
         client.admin().indices().analyze(analyzeRequest, new RestToXContentListener<AnalyzeResponse>(channel));
     }
+
+    public static void parseContent(AnalyzeRequest request, String text, BytesReference content) {
```

Comment:
@martijnvg Question about BWC.

Your comment means that we remove other fields from AnalyzeRequest and add a source field, right?
If we do that, I think we will break the backward compatibility.
Is it correct?